src/registries/npm-registry.js

```diff
@@ -72,9 +73,39 @@ export default class NpmRegistry extends Registry {
     return name.replace('/', '%2f');
   }
 
+  getRequestUrl(registry: string, pathname: string): string {
+    const isUrl = /^https?:/.test(pathname);
```

Comment:
Nitpick: technically `git://xxx.com/yyy` is also a URL so you'd want to name this as `isHTTPUrl` or something.

Reply:
This will break protocol relative URLs, which are generated by default in `.npmrc`

Reply:
Oh. Issue filed somewhere?

Reply:
https://github.com/yarnpkg/yarn/pull/4347Oh. Issue filed somewhere?